api/src/pcapi/alembic/versions/20240105T125353_10b54017d02a_delete_providablemixin_on_venueprovider.py

```diff
@@ -0,0 +1,44 @@
+"""Delete providable mixin from venue provider"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "10b54017d02a"
+down_revision = "cbd4d45e39f0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.drop_column("venue_provider", "dateModifiedAtLastProvider")
+    op.drop_column("venue_provider", "lastProviderId")
+    op.drop_column("venue_provider", "fieldsUpdated")
+    op.drop_column("venue_provider", "idAtProviders")
+
+
+def downgrade() -> None:
+    op.add_column(
+        "venue_provider", sa.Column("idAtProviders", sa.VARCHAR(length=70), autoincrement=False, nullable=True)
+    )
+    op.add_column(
+        "venue_provider",
+        sa.Column(
+            "fieldsUpdated",
+            postgresql.ARRAY(sa.VARCHAR(length=100)),
+            server_default=sa.text("'{}'::character varying[]"),
+            autoincrement=False,
+            nullable=True,
```

Comment:
Elle est non nullable à la base 
```suggestion
            nullable=False,
```

Reply:
Nous ne pouvons pas ajouter une colnne en nullable=False comme ça ([voir ce fichier](https://github.com/pass-culture/pass-culture-main/blob/26ae5fcc2eeaadc280f6128e0deebc9a492b6345/api/bin/alembic_add_not_null_constraint.py#L2) qui permet de générer des migrations pour rendre une colonne not null. Dans notre cas cette colonne est pas utilisée. On fait rarement de downgrade mais si on en fait un ça ne serait pas très grave d'avoir ce décalage

Reply:
(du coup je corrige le commentaire)